sonar-plugin-api/src/main/java/org/sonar/api/batch/sensor/cpd/internal/DefaultCpdTokens.java

```diff
@@ -87,7 +88,7 @@ public DefaultCpdTokens addToken(TextRange range, String image) {
     if (excluded) {
       return this;
     }
-    Preconditions.checkState(lastRange == null || lastRange.end().compareTo(range.start()) <= 0,
+    checkState(lastRange == null || lastRange.end().compareTo(range.start()) <= 0,
```

Comment:
This is my next proposal, that's why I didn't touch any of these Preconditions uses ! I suggest to create `org.sonar.api.utils.MoreUtils` with simple methods like checkState(), checkArgument() or isEmpty(), like what was done here : https://github.com/SonarSource/orchestrator/blob/master/sonar-orchestrator/src/main/java/com/sonar/orchestrator/util/OrchestratorUtils.java